src/internal/connector/onedrive/drive.go

```diff
@@ -303,7 +303,7 @@ func getFolder(
 }
 
 // Create a new item in the specified folder
-func createItem(
+func CreateItem(
```

Comment:
nbd for this PR, but since we're exporting it, would be great to move this into the `api` as a follow-up.